docs/guide/database-queries.md

```diff
@@ -0,0 +1,273 @@
+  .toObject();
+```
+
+### `toCount`
+
+The `toCount` query method is the easy and quickest way to simply get the number of documents that match a query. The `toCount` query method will ignore any `page`, `skip` and `limit` query builders in order to get the total number.
+
+If the query is using a `perPage` query builder it will also populate `totalPages` on the query, which can be retrieved with `query.get('totalPages')`.
+
+```javascript
+const productsCount = await self.find(req, criteria)
+  .toCount();
+```
+
+### `toDistinct`
+
+`toDistinct` allows us to retrieve the unique values for a particular document property from the documents that match query. For example, using `query.toDistinct('category')` will return an array with all the `category` property values across documents, with each category only appearing once in the array.
+
+```javascript
+const shoeColors = await self.find(req, { type: 'shoe' })
+  .toDistinct('color');
+```
+
+### `toChoices`
+
+The `toChoices` query method builds on `toDistinct` by returning each choice as an object with `label` and `value` properties. This can be useful when populating a select or other input field with options for a doc type property. For example, this is used for the document manager modal filter UI in Apostrophe.
+
+`toChoices` accepts an options object as an optional second argument. Setting `counts: true` in that options object will include a `count` property on each returned choice indicating how many documents match that choice.
+
+```javascript
+const teamOffices = await self.find(req, { type: 'team' })
+  .toDistinct('office', { count: true });
+```
+
+## Query across modules
+
+The `find()` method in doc type modules is easy to use within each module as `self.find`. However this assumes we're looking for only content that is governed by the module where the method is called. There are two main ways to write these database queries from one module and get content managed by a separate module.
+
+### Using another module's `find` method
+
+The `self` object available in any Apostrophe module's customization functions can access other module managers on the `self.apos.modules` object. For example, if I'm working in the `article` module and want to query `author` pieces, I can access the `author` module with this:
+
+```javascript
+self.apos.modules.author
+```
+
+`self.apos.modules.author` is essentially the same as `self` would be if we were operating within the `author` module. Therefore we can query author pieces directly by using that module's `find` method.
+
+```javascript
+const activeAuthors = self.apos.modules.author.find(req, { active: true })
+  .toArray();
+```
+
+If a doc type module has been [assigned an alias](/reference/module-api/module-options.md#alias), the doc type's manager will be directly on `self.apos`. For example, the `@apostrophecms/page` module is available as `self.apos.page` because `'page'` is its alias option value.
```

Comment:
How about "then that doc type module will be..."

Manager is different language than the beginning of the sentence and probably doesn't need to be brought up now that the module name and the doc `type` in the database are always equal.